lib/Element.js

```diff
@@ -88,6 +88,22 @@ Element.prototype = Object.create(ContainerNode.prototype, {
       }
     }
   },
+  innerText: {
+    get: function() {
+      var strings = [];
+      recursiveGetText(this, strings);
+      // Strip and collapse whitespace
+      // This doesn't 100% match the browser behavior
```

Comment:
nit: just a tiny proposal to add a bit more context here.
```suggestion
      // This doesn't 100% match the browser behavior,
      // but should cover most of the cases. This is also similar to
      // how Angular's renderer used to work: the `textContent` and `innerText`
      // were almost equivalent from the renderer perspective.
```